### PR TITLE
tiltfile: document quiet flag for local()

### PR DESCRIPTION
**Pull request by tmc**
This reflects https://github.com/windmilleng/tilt/pull/2808

### CODE REVIEW

File: api/api.py [hunk 1]
```diff
@@ -312,8 +312,9 @@ def load(path: str, *args):
     create_namespace('frontend')
   """
 
-def local(cmd: str) -> Blob:
-  """Runs cmd, waits for it to finish, and returns its stdout as a ``Blob``."""
+def local(cmd: str, quiet: bool = False) -> Blob:
+  """Runs cmd, waits for it to finish, and returns its stdout as a ``Blob``. If ``quiet`` is set to True, skips omitting output.
```
Comment (jazzdan) [inline api/api.py]: Now that this command has two arguments, should we reformat it to look like the others with an arglist? e.g.

```python
def local(cmd: str, quiet: bool = False) -> Blob:
  """Runs cmd, waits for it to finish, and returns its stdout as a ``Blob``

  Args:
    cmd: Command to run
    quiet: If set to True, skips printing output to log
  """
```

Comment (jazzdan) [reply api/api.py]: I also think the "skips omitting output" verbiage is confusing, so I tried rewording it in the example above to: "skips printing output to log".

Comment (tmc) [reply api/api.py]: Yeah I meant to write emitting. I’ll clean up the signature and the language.

Comment (tmc) [reply api/api.py]: Yeah I meant to write emitting. I’ll clean up the signature and the language.

Comment (tmc) [reply api/api.py]: Yeah I meant to write emitting. I’ll clean up the signature and the language.

**Comment by tmc** [comment]
@jazzdan PTAL.

**Comment by jazzdan** [comment]
Thanks @tmc!

**Pull request merged**